plugins/catalog-backend/src/modules/core/PlaceholderProcessor.ts

```diff
@@ -127,6 +134,8 @@ export class PlaceholderProcessor implements CatalogProcessor {
           base,
         });
 
+      emit(processingResult.refresh(`url:${resolverValue}`));
```

Comment:
Not actually the correct value here right? It should be the absolute resolved target URL. Looking like this would be the relative one

Reply:
yea I think you are correct here, this will be the relative one